### ISSUE TITLE
Reevaluate the date picker implementation

### ISSUE THREAD

**Issue by Windvis** [appuniversum/ember-appuniversum#402]
Do we still need a custom date picker library for this. Maybe the native one is good enough for our use cases now? (ignoring range pickers etc, which would probably be a different component anyways).

**Comment by Windvis** [comment]
Issues I noticed with the native one:
- the browser calendar icon can't be hidden in Firefox. This might be confusing if we also show the Appuniversum icon in there
- The implementation is browser specific and some of them have some weird UX. Not sure if this is an issue for users
- no control over the date formatting. It selects one based on your browser language which might not match the one used in other parts of the app

**Comment by aliokan** [comment]
@Windvis Any plan to move forward on this topic?

**Comment by Windvis** [comment]
Not really. The current date picker project seems dead, but the native one doesn't seem ideal either, from what I can read in my previous comment. Not sure what to do with this, other than researching other date picker libraries maybe. But it's not a blocker for a project that I work on, so I haven't looked into it.

Is there anything specific you want to see?
